test/rubocop/cop/magic_numbers/no_magic_numbers_test.rb

```diff
@@ -241,46 +241,6 @@ def test_method
           )
         end
 
-        def test_ignores_magic_integers_as_arguments_to_methods_on_another_object
```

Comment:
Is this intentional?

Reply:
Fixed in 4337f731f64eb71c21f319c217874c7de8984cb5